src/dev/impl/DevToys/Strings/en-US/PasswordGenerator.resw

```diff
@@ -0,0 +1,168 @@
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="AccessibleName" xml:space="preserve">
+    <value>Password Generator tool</value>
+  </data>
+  <data name="Configuration" xml:space="preserve">
+    <value>Configuration</value>
+  </data>
+  <data name="MenuDisplayName" xml:space="preserve">
+    <value>Password Generator</value>
+  </data>
+  <data name="Generate" xml:space="preserve">
+    <value>Generate</value>
+  </data>
+  <data name="GenerateButton" xml:space="preserve">
+    <value>Generate Passwords(s)</value>
+  </data>
+  <data name="Passwords" xml:space="preserve">
+    <value>Passwords</value>
+  </data>
+  <data name="MultiplySymbol" xml:space="preserve">
+    <value>x</value>
+  </data>
+  <data name="NumberOfPasswordsToGenerate_AutomationProperties_Name" xml:space="preserve">
+    <value>Number of Passwords to generate</value>
+  </data>
+  <data name="Uppercase" xml:space="preserve">
+    <value>Include Lowercase Characters in a Password (ABCDEFGHJKLMNOPQRSTUVWXYZ)</value>
+  </data>
+  <data name="Description" xml:space="preserve">
+    <value>Generate random passwords</value>
+  </data>
+  <data name="Digits" xml:space="preserve">
```

Comment:
I don't see where the Digits text is displayed. More generally, it looks like there's no "Digits" option in the UI. Should this be removed?

Reply:
Please see this comment:
https://github.com/veler/DevToys/pull/664/files#r975456968